practice/README.md

```diff
@@ -0,0 +1,57 @@
+# React Practice
+
+## OVERVIEW
+
+- This document provides requirements, technical stack, and estimation for React Practice.
+
+## TIMELINE
+
+- Expected: 32hours  (from 2024/05/22 - 2024/05/30)
+
+## TEAM SIZE
+
+- One developer.
+
+## TARGET
```

Comment:
These targets are not for practice 1 React, we focus on building components and using storybook. Please check again